pkg/replication/order_test.go

```diff
@@ -0,0 +1,45 @@
+package replication
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/anthonybishopric/gotcha"
+)
+
+func TestMaxOrderBlocks(t *testing.T) {
+	order, err := NewMaxOrder([]string{"a", "b", "c", "d", "e"}, 2, nil)
+	Assert(t).IsNil(err, "should have been nil error")
+	next := make(chan string)
+	done := make(chan string)
+	errs := make(chan error)
+	quit := make(chan struct{})
+
+	go order.Execute(next, done, errs, quit)
+
+	count := 0
+OUTER:
+	for {
+		select {
+		case <-next:
+			count++
+		case <-time.After(500 * time.Millisecond):
```

Comment:
Is there any way to rewrite this loop such that it doesn't guarantee an added 500ms to the test suite?